pkg/networkservice/common/heal/client.go

```diff
@@ -66,69 +70,97 @@ func NewClient(client networkservice.MonitorConnectionClient, onHeal *networkser
 		requestors:        make(map[string]func()),
 		closers:           make(map[string]func()),
 		reported:          make(map[string]*networkservice.Connection),
+		connectionIDs:     make(map[string]bool),
```

Comment:
What are we trying to do with this field?  I see it set, but never really read.

Reply:
Yes, old one, will fix

Reply:
I figured :)  Many thanks :)